versioned_docs/version-v3.0.0/node/node-linux-packages.md

```
@@ -0,0 +1,299 @@
+
+:::
+
+When installing HOPRd via native packages on Linux (e.g., `.deb` or `.rpm`), the node integrates directly into the Linux file system using standard conventions.
+
+---
+
+## Determine machine architecture
+
+In your terminal window, run the following command to determine your machine’s architecture: 
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)